### PR TITLE
Fix migration issue

### PR DESCRIPTION
**Pull request by harshkothari410**
Here I removed the checkDB functionality into postinstall and added to prestart. So that there is no issue of private space heroku migration / resetdb issue.

There will be little bit overhead on npm start but it results in lots of simplicity in automatic migration and deployment of new refocus on any platform.

### CODE REVIEW

File: db/createOrUpdateDb.js [hunk 1]
```diff
@@ -18,38 +18,19 @@ const models = require('./index');
...
-    'Heroku Private Space');
-  process.exit(0); // eslint-disable-line
-}
-
-console.log('querying information_schema.tables where table_schema = \'public\'')
 models.sequelize.query(`select count(*) from
   information_schema.tables where table_schema = 'public'`)
 .then((data) => {
-  console.log('found', data)
+  console.log('found', data); // eslint-disable-line
```
Comment (iamigo) [inline db/createOrUpdateDb.js]: I added this when troubleshooting migration issues a while back. We don't need to keep this console.log anymore either, unless it's useful to you for migration troubleshooting

File: db/createOrUpdateDb.js [hunk 2]
```diff
@@ -18,38 +18,19 @@ const models = require('./index');
...
-  if (err.name === 'SequelizeConnectionError' &&
-    err.message.startsWith('connect ETIMEDOUT')) {
-    console.log('Exiting "./db/createOrUpdateDb"... ' +
-      'SequelizeConnectionError: connect ETIMEDOUT');
-    process.exit(0); // eslint-disable-line
-  }
-
   const dbConfig = utils.dbConfigObjectFromDbURL();
-  console.log('create db now', dbConfig)
+  console.log('create db now', dbConfig); // eslint-disable-line
```
Comment (iamigo) [inline db/createOrUpdateDb.js]: we don't need to keep this console.log anymore either, unless it's useful to you for migration troubleshooting

**Comment by iamigo** [comment]
@harshkothari410 does this fix the migration issues @meynet hit in heroku?

**Comment by harshkothari410** [comment]
Yeah I tested on Heroku so I am assuming that it is fixed.

File: db/createOrUpdateDb.js (hunk 2 above)
Comment (harshkothari410) [inline db/createOrUpdateDb.js]: Okies sure I am removing this.

File: db/createOrUpdateDb.js (hunk 1 above)
Comment (harshkothari410) [inline db/createOrUpdateDb.js]: Removing this as well.

**Comment by pallavi2209** [comment]
@harshkothari410 Resolve conflicts please.
Not related to this PR, but just checking, can we run sequelize.sync({ force: false }) now without problems (related to: when Ian ran into problems when he wanted to run db sync for new table creation) @iamigo

**Comment by harshkothari410** [comment]
Is there any specific script we have for sequelize.sync({ force: false }) ? @pallavi2209

**Comment by iamigo** [comment]
No. We need to add a call to sequelize sync from the doImport function in db/index.js.

**Comment by pallavi2209** [comment]
Nevermind, I noticed a related story on Ian's dashboard for this sprint. Can you resolve conflicts in package.json for this PR? @harshkothari410

**Comment by iamigo** [comment]
@harshkothari410 please let me know if you need help resolving the package.json conflicts so we can get this merged.

**Pull request merged**